runelite-client/src/main/java/net/runelite/client/plugins/devtools/DevToolsOverlay.java

```diff
@@ -437,9 +444,6 @@ private void renderGraphicsObjects(Graphics2D graphics)
 
 	public void renderWidgets(Graphics2D graphics)
```

Comment:
This can be `private`

```suggestion
	private void renderWidgets(Graphics2D graphics)
```

Reply:
Also, almost all of the `public void renderObjects(Graphics2D graphics)` methods can become `static`. Looks like only the following can't become static:

* renderPlayers
* renderNpcs
* renderTileObjects
* renderTileTooltip
* renderInventory
* renderProjectiles
* renderGraphicsObjects

Reply:
I dont really want to start making stuff static, or at least not in PR